[master] Update for cc Enforcer's dropins folder with helm artifacts (#6415)
wso2/docs-apim: en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/configure-analytics.md

     
    1. Compile the new custom reporter implementation as a JAR file.</br>
    2. Mount it to the `/home/wso2/lib/dropins` directory within the Enforcer by adding the JAR file to the choreo-connect-1.x.x/docker-compose/resources/enforcer/dropins directory.</br>

        !!! note 

>             If you use Choreo Connect with Helm Charts, please refer to the documentation in [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-files-into-the-dropins-directory-optional) to add a JAR file into the dropins directory.

Reviewer: ```suggestion
            If you use Choreo Connect with Helm Charts, please refer to the documentation [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-files-into-the-dropins-directory-optional) to add a JAR file into the dropins directory.
```

>             If you use Choreo is Connect with Helm Charts, please refer to the documentation in [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-files-into-the-dropins-directory-optional) to add a JAR file into the Droppings directory.

Reviewer 2: This is in other places too
```suggestion
            If you use Choreo Connect with Helm Charts, please refer to the documentation in [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-files-into-the-dropins-directory-optional) to add a JAR file into the Droppings directory.
```

>             If Choreo is Connect deployed using the Kubernetes Helm artifacts, dropins folder mounting to the Enforcer is explained in [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-dropins-directory-to-a-choreo-connect-helm-deployment)

Reviewer 2: ```suggestion
            Helm Chart users, please refer to the documentation [here]({{base_path}}/deploy-and-publish/deploy-on-gateway/choreo-connect/production-deployment-guideline/#mount-dropins-directory-to-a-choreo-connect-helm-deployment) to add a JAR file into the Droppings directory.
```

Reviewer 2: Please use the updated doc link if the doc header is updated with the other comments. I mean `#mount-dropins-directory-to-a-choreo-connect-helm-deployment` part of the link
